### PR TITLE
Update plugin.js

### PR DESCRIPTION
**Pull request by robcarey**
Added context menu support

### CODE REVIEW

File: src/plugin.js [hunk 1]
```diff
@@ -312,6 +312,25 @@
             onAction: startPlugin
         });
 
+        const setupMenuItem = editor => {
```
Comment (prathamVaidya) [inline src/plugin.js:315]: No need to surround the code in function. Add them directly like the `addButton`.

**Review by prathamVaidya — CHANGES_REQUESTED** [review]

**Comment by prathamVaidya** [comment]
@robcarey You have made the changes to `main`. The latest modal feature is in the branch `feat-modal-fallback`. You have to use that branch and create a PR against that branch only. Thanks

**Comment by robcarey** [comment]
@prathamVaidya I'm not sure how to delete this.   I'm not super familiar with github to be honest

**Comment by prathamVaidya** [comment]
@robcarey no worries. I can help.

You first switch to the feat-modal-fallback branch and then edit the code. Then commit and push. Then create a new PR. In the new PR option you will see option against which branch you want to create it. Choose the correct feat-modal-fallback branch. And create it.

In case you don't have time or don't want to do I can copy and paste the code from my side. But I'd recommend you do. 😄

**Pull request closed without merging**